introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs

```diff
@@ -549,3 +560,81 @@ pub(crate) fn primary_key_is_clustered(
 
     Some(ext.pk_is_clustered(table_id))
 }
+
+fn get_opclass(
+    index_field_id: sql::IndexFieldId,
+    schema: &SqlSchema,
+    ctx: &IntrospectionContext,
+) -> Option<OperatorClass> {
+    if !ctx.sql_family().is_postgres() {
+        return None;
+    }
+
+    let ext: &PostgresSchemaExt = schema.downcast_connector_data();
+
+    let opclass = match ext.get_opclass(index_field_id) {
```

Comment:
`ext.get_opclass(index_field_id)?` and you can get rid of the match.

Reply:
No, because the return value of this function is `()`.